temba/orgs/models.py

```diff
@@ -2008,6 +2010,9 @@ def delete(self):
 
         for group in self.all_groups.all():
             ContactGroupCount.objects.filter(group=group).delete()
+
+            ContactImport.objects.filter(group=group).delete()
```

Comment:
This line throws an error in the unit test and wants `ContactImportBatch` to be deleted first.